lib/internal/bootstrap/node.js

```diff
@@ -114,6 +114,11 @@ if (isMainThread) {
   process.memoryUsage = wrapped.memoryUsage;
   process.kill = wrapped.kill;
   process.exit = wrapped.exit;
+
+  process._getSourceMapCache = function() {
```

Comment:
Generally not a fan of adding new `_` prefixed methods to `process`. If this is not intended to be public API, then it could go into `internal/util`, otherwise, just make it a public API.

Reply:
I was leaning towards it being public for tool makers, just drop the _ prefix?

Reply:
Yeah, I would drop the prefix and would also really consider whether it makes sense to have on `process` or not. Is any part of the specific to v8? If so, it might make more sense to put on that module.

Reply:
@jasnell I could imagine that we eventually pass some of the source-map information to V8, if V8 eventually supports some of this functionality itself for stack traces. I think a good chunk of the implementation will live in Node.js for internal and external tooling, however -- unlike test coverage, where we are leaning directly on V8 for the brunt of the functionality.

Is there another `.js` module I can crib, from which we interact with through `C++`; the process object was convenient because we already interact with this object in both contexts (doesn't seem like a pattern we use too often in the codebase, I believe we callout from JavaScript to C++ bindings more often than from C++ to a JavaScript object).

Reply:
At this point, I'm thinking either `util` or `v8` would be a better place to expose this than on the `process` object. I'm thinking that it should be initially marked as experimental to give us the freedom to iterate on it a bit more before it falls under official semver support.

Reply:
Also, there will need to be some documentation for this added if it's going to be public

Reply:
I don’t really think this is resolved at this point … `util` seems like a good place to me.